### PR TITLE
Add Directory.Build.props

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: lib/PuppeteerSharp/PuppeteerSharp.csproj [hunk 1]
```diff
@@ -39,11 +39,7 @@
     <CodeAnalysisRuleSet>../PuppeteerSharp.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
   <PropertyGroup>
-    <DocumentationFile>bin\$(Configuration)\netstandard2.0\PuppeteerSharp.xml</DocumentationFile>
```
Comment (SimonCropp) [inline lib/PuppeteerSharp/PuppeteerSharp.csproj:42]: no need for full path here. GenerateDocumentationFile sorts that for you. and will also work if u ever decide to multitarget frameworks

File: lib/PuppeteerSharp.sln [hunk 2]
```diff
@@ -1,22 +1,23 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
```
Comment (SimonCropp) [inline lib/PuppeteerSharp.sln:3]: let me know if u want the sln reverted back to VS15

Comment (kblok) [reply lib/PuppeteerSharp.sln:3]: This is perfect!

File: samples/get-all-links/get-all-links.csproj [hunk 3]
```diff
@@ -4,7 +4,6 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <RootNamespace>Example</RootNamespace>
-    <LangVersion>7.1</LangVersion>
```
Comment (kblok) [inline samples/get-all-links/get-all-links.csproj:7]: These sample projects are not part of the solution. How about setting the LangVersion to 8 in these projects?

Comment (SimonCropp) [reply samples/get-all-links/get-all-links.csproj:7]: VS walks the dir tree for Directory.Build.props, which is at the root, so it covers all the projects

Comment (SimonCropp) [reply samples/get-all-links/get-all-links.csproj:7]: https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019#search-scope

**Pull request merged**